channeldb/payments.go

```diff
@@ -28,20 +27,35 @@ var (
 	// root-bucket
 	//      |
 	//      |-- <paymenthash>
-	//      |        |--sequence-key: <sequence number>
-	//      |        |--creation-info-key: <creation info>
-	//      |        |--attempt-info-key: <attempt info>
-	//      |        |--settle-info-key: <settle info>
-	//      |        |--fail-info-key: <fail info>
+	//      |        |--mpp-sequence-key: <sequence number>
```

Comment:
why the prefix `mpp` actually? it is the only type we support in the db? or is it a prep. for amp?

Reply:
I added it to distinguish it from the old key, but we can rename it when we know what the final structure should be.